operator/pkg/manifest/shared.go

```diff
@@ -441,9 +441,30 @@ func getClusterSpecificValues(client kube.Client, force bool, l clog.Logger) (st
 	if fsgroup != "" {
 		overlays = append(overlays, fsgroup)
 	}
+	cni := getCNISettings(client)
+	if cni != "" {
+		overlays = append(overlays, cni)
+	}
 	return makeTreeFromSetList(overlays)
 }
 
+// getCNISettings gets auto-detected values based on the Kubernetes environment.
+// Note: there are other settings as well; however, these are detect inline in the helm chart.
+// This ensures helm users also get them.
```

Comment:
From the previous issue with fsGroup injection, this call hierarchy was basically for operator and istioctl install ? So I got a little confused with this comment.

Reply:
Helm users MUST explicitly set namespace; in istioctl they do not need to. So this just detects namespace.

Reply:
Okay, thanks for the clarification.